lib/organization/role.ex

```diff
@@ -0,0 +1,43 @@
+defmodule GlassFactoryApi.Organization.Role do
+  @moduledoc """
+  Defines a Role of an Organization.
+  """
+
+  @type t() :: %__MODULE__{
+          id: String.t(),
+          name: String.t(),
+          description: String.t(),
+          department_id: String.t(),
+          is_used: boolean(),
+          rate: float(),
+          cost: float(),
```

Comment:
Even if the API serializes this as a float, I'd strongly advise [against using floats](http://blog.plataformatec.com.br/2014/09/floating-point-and-currency/) to represent currency values. Could we force this field to be a String instead?